### ISSUE TITLE
BUG: Build in Bandwidth limiter not working on local bridge setup

### ISSUE THREAD

**Issue by karhouchai** [SoftEtherVPN/SoftEtherVPN#1488]
### Prerequisites

* [x] Can you reproduce? 
* [x] Are you running the latest version of SoftEtherVPN?

**SoftEther version:** 4.38 Build 9760
**Component:** Server
**Operating system:** Linux Ubuntu 16.04
**Architecture:**  64bit

**Processor:** KVM VPS

### Description
I have an issue with Softether server's build in bandwidth limiter that can be configured under "Groups" -> "Security Policy" -> "Download Bandwidth". It seems like its not working with local bridge setup with dnsmasq. This feature is working fine with build in SecureNAT.  I have checked if my local bridge setup is ok and I can get correct VPS IP on my computer.  Previously I'm using Server with Ver 4.2 Build 9608, the build in bandwidth limiter seems to work fine with local bridge. Both old server build and latest server build I'm using same method to configure the local bridge 

**Expected behavior:**
Build in bandwidth limiter would work with local bridge setup.

**Actual behavior:**
Build in bandwidth limiter would not work with local bridge setup. SecureNAT is fine.

### Steps to reproduce

1. Check client external IP without connecting to VPN [i.e. It shows ISP IP]
2. Set Download Bandwidth in security policy [Set 30Mbps]
3. Disable Dnsmasq and delete  local bridge tap driver from softether server
4. Enable SecureNAT and connect to VPN from client
5. Check client external IP  [i.e. It shows VPS IP]
6. Run Speed test [Cap at 30Mbps, this shows SecureNAT is working fine with build in bandwidth limiter]
7. Disconnect VPN and check external IP [i.e. It shows back ISP IP which is correct]
8. Disable SecureNAT, enable Dnsmasq and add local bridge tap driver in softether server
9. Connect to VPN and check client external IP [i.e it shows VPS IP, means settings is correct and routed correctly]
10. Run speed test [ Speed at 100Mbps, maximum limit of VPS]

Not sure if others are seeing this issue as well on the latest SofEther build.

**Comment by domosekai** [comment]
Did you restart the server when switched from secure NAT to local bridge?

**Comment by domosekai** [comment]
I can't reproduce the issue. Bandwidth limit works fine in local bridge.

**Comment by karhouchai** [comment]
@domosekai Thanks for trying to reproducing this.

Yes, I did restart the server once switch from secure NAT to local bridge.
Anyhow I manage to found out why the bandwidth limit does not work on my end.
Its because I did not configure my ipv6 correct. The story goes like this.
The VPN hosted at my VPS are only assigned with ipv4 only and I choose to did not configure anything related with ipv6 in dnsmasq, whereas my client connects to WAN and are assigned with both IPV4 and IPV6. So what I did at my end was once I disable the WAN IPv6 at client side, bandwidth limiters works perfectly fine with local bridge.

Thus I believe this not not a valid bug and more towards my misconfiguration. I will further debug on my end.
@domosekai , once again I apologize for wasting your effort to debug this and I will close the ticket.
